config/solr_query_config.yml

```diff
@@ -74,7 +73,6 @@ NSIDC:
     title: 25
     summary: 25
     parameters: 5
-    keywords: 5
```

Comment:
I can't figure out a way to actually put a comment at line 71/72, so adding it here: Should "keywords" be removed at line 71/72?

Reply:
Same question for line 118/115 -- although I just realized that this story is only for removing the boosts, not the keywords themselves, so perhaps the remaining "keyword" entries are ok.